examples/Cavity2D/CSMdummy.py

```diff
@@ -1,25 +1,21 @@
 from __future__ import division
-
 import numpy as np
 import precice
+from mpi4py import MPI
 
 configuration_file_name = "precice-config.xml"
 participant_name = "CSMdummy"
 mesh_name = "beam"
 
-
 def computeDisplacements(force_vals, displ_vals, coords_x):
     print("Entering computeDisplacements")
-
-    # Considering beam of square cross-section with thickness of 0.01 m
-    a = 0.01
-    # Area moment of inertia about neutral axis of square cross-section
-    MI = pow(a, 4) / 12
-    # Modulus of elasticity of copper = 117 GPa
-    ME = 117 * pow(10, 9)
-    # Length of beam = 1 m as stated in Fluent
-    ll = 1
-
+    # Beam properties
+    a = 0.01              # Thickness [m]
+    MI = a**4 / 12        # Area moment of inertia
+    ME = 117e9            # Modulus of elasticity (117 GPa)
+    ll = 1.0              # Beam length [m]
```

Comment:
```suggestion
    a = 0.01              # Thickness [m]
    MI = a**4 / 12        # Area moment of inertia of square cross section
    ME = 117e9            # Modulus of elasticity (117 GPa) of copper
    ll = 1.0              # Beam length [m]
```
better keep this information. Otherwise these number are just "magic" numbers.